static/app/views/alerts/rules/metric/triggers/chart/index.tsx

```diff
@@ -134,6 +134,11 @@ const TIME_WINDOW_TO_SESSION_INTERVAL = {
   [TimeWindow.ONE_DAY]: '1d',
 };
 
+const EXTRAPOLATED_SESSION_AGGREGATE_TO_HEADING = {
+  [SessionsAggregate.CRASH_FREE_SESSIONS]: t('Estimated Sessions'),
+  [SessionsAggregate.CRASH_FREE_USERS]: t('Estimated Users'),
+};
```

Comment:
@matejminar Not sure if those would ever be used?
But it should not hurt if they are there 🤷

Reply:
on-demand metrics extraction is not for sessions so let's remove it to not cause confusion 👍